src/components/shared/SgtNavigationDrawer/SgtNavigationDrawer.vue

```diff
@@ -57,111 +57,72 @@
               <v-list-item-title
                 :id="navItem.title"
                 :class="{ 'font-grey': !$route.path.includes(navItem.path) }"
-                >{{ $t(navItem.title) }}</v-list-item-title
-              >
+              >{{ $t(navItem.title) }}</v-list-item-title>
             </v-list-item>
           </template>
           <span>{{ $t(navItem.title) }}</span>
         </v-tooltip>
 
         <v-list-item class="nav-brand">
           <v-list-item-icon class="nav-logo">
-            <img src="@/assets/images/seagate-green.svg" alt="logo" />
+            <img v-if="brandLogo" :src="require(`@/assets/${brandLogo}`)" alt="logo" />
           </v-list-item-icon>
-          <v-list-item-title class="logo-title">Seagate</v-list-item-title>
+          <v-list-item-title class="logo-title">{{brandName}}</v-list-item-title>
         </v-list-item>
       </v-list>
     </v-navigation-drawer>
   </div>
 </template>
 
 <script lang="ts">
-import { Component, Vue, PropSync } from "vue-property-decorator";
-import router from "../router";
+import { Component, Vue, PropSync, Prop } from "vue-property-decorator";
+import router from "@/router";
+import { SgtNavigationDrawerItem } from "./SgtNavigationDrawerItem.model";
 
 @Component({
-  name: "LrNavigationDrawer",
-  components: {},
+	name: "SgtNavigationDrawer",
+	components: {},
 })
 export default class LrNavigationDrawer extends Vue {
-  @PropSync("drawer", { type: Boolean }) syncedName!: boolean;
-  activeRoute = "";
-  navItems = [
-    {
-      title: "dashboard",
-      path: "/dashboard",
-      iconDefault: "dashboard-grey.svg",
-      iconActive: "dashboard-white.svg",
-      requiredAccess: "alerts",
-    },
-    {
-      title: "health",
-      path: "/health",
-      iconDefault: "health-grey.svg",
-      iconActive: "health-white.svg",
-      requiredAccess: "sysconfig",
-    },
-    {
-      title: "manage",
-      path: "/manage",
-      iconDefault: "manage-grey.svg",
-      iconActive: "manage-white.svg",
-      requiredAccess: "s3accounts",
-    },
-    {
-      title: "lyvePilot",
-      path: "/ldp",
-      iconDefault: "udx-grey.svg",
-      iconActive: "udx-white.svg",
-      requiredAccess: "lyve_pilot",
-    },
-    {
-      title: "settings",
-      path: "/settings",
-      iconDefault: "settings-grey.svg",
-      iconActive: "settings-white.svg",
-      requiredAccess: "maintenance",
-    },
-    {
-      title: "maintenance",
-      path: "/maintenance",
-      iconDefault: "maintenance-grey.svg",
-      iconActive: "maintenance-white.svg",
-      requiredAccess: "sysconfig",
-    },
-  ];
+	@PropSync("drawer", { type: Boolean }) syncedName!: boolean;
+	@Prop({ required: true }) navItems: SgtNavigationDrawerItem[];
+	@Prop({ required: false }) brandLogo: string;
+	@Prop({ required: false }) brandName: string;
```

Comment:
brandName prop is optional and is required on line 70.

Reply:
fixed